src/gui/tray/Window.qml

```diff
@@ -271,51 +283,9 @@ Window {
                         }
                     }
 
-                    background:
-                        Item {
-                        id: leftHoverContainer
-
-                        height: Style.trayWindowHeaderHeight
-                        width:  Style.currentAccountButtonWidth
-                        Rectangle {
-                            width: Style.currentAccountButtonWidth / 2
-                            height: Style.trayWindowHeaderHeight / 2
-                            color: "transparent"
-                            clip: true
-                            Rectangle {
-                                width: Style.currentAccountButtonWidth
-                                height: Style.trayWindowHeaderHeight
-                                radius: Style.trayWindowRadius
-                                color: "white"
-                                opacity: 0.2
-                                visible: accountBtnMouseArea.containsMouse
-                            }
-                        }
-                        Rectangle {
-                            width: Style.currentAccountButtonWidth / 2
-                            height: Style.trayWindowHeaderHeight / 2
-                            anchors.bottom: leftHoverContainer.bottom
-                            color: "white"
-                            opacity: 0.2
-                            visible: accountBtnMouseArea.containsMouse
-                        }
-                        Rectangle {
-                            width: Style.currentAccountButtonWidth / 2
-                            height: Style.trayWindowHeaderHeight / 2
-                            anchors.right: leftHoverContainer.right
-                            color: "white"
-                            opacity: 0.2
-                            visible: accountBtnMouseArea.containsMouse
-                        }
-                        Rectangle {
-                            width: Style.currentAccountButtonWidth / 2
-                            height: Style.trayWindowHeaderHeight / 2
-                            anchors.right: leftHoverContainer.right
-                            anchors.bottom: leftHoverContainer.bottom
-                            color: "white"
-                            opacity: 0.2
-                            visible: accountBtnMouseArea.containsMouse
-                        }
+                    background: Rectangle {
+                        color: currentAccountButton.hovered ? "white" : "transparent"

```

Comment:
I would expect the hover effect to be broken. So I think that either you want to add `hoverEnabled: Style.hoverEffectsEnabled` in `currentAccountButton` or you need to use `accountBtnMouseArea.containsMouse`.

Reply:
I was expecting the same but it is working:
![hover](https://user-images.githubusercontent.com/241266/88566136-e0b4a280-d035-11ea-99ba-1cb16fd2daee.png)
![hover](https://user-images.githubusercontent.com/241266/88566390-38eba480-d036-11ea-988f-0df923904a04.png)

Reply:
Changed it.

Reply:
> I was expecting the same but it is working:
> ![hover](https://user-images.githubusercontent.com/241266/88566136-e0b4a280-d035-11ea-99ba-1cb16fd2daee.png)
> 
> Should I change yet?

Odd, was failing for me. I wonder if it's a platform thing with hoverEnabled default being different on the Button...
:hourglass_flowing_sand:
Alright, after a quick check in QtQuick2 and QtQuickControls2 code... hoverEnabled is indeed following `QGuiApplication::styleHints()->useHoverEffects()` by default.